mantidimaging/gui/windows/live_viewer/presenter.py

```diff
@@ -30,3 +32,31 @@ def __init__(self, view: LiveViewerWindowView, main_window: MainWindowView):
         self.view = view
         self.main_window = main_window
         self.model = LiveViewerWindowModel(self)
+
+    def set_dataset_path(self, path):
+        """Set the path to the dataset."""
+        self.model.path = path
+
+    def update_image(self, image_path: str):
+        """Update the image in the view."""
+        if not Path(image_path).exists():
+            return
+        try:
+            with tifffile.TiffFile(image_path) as tif:
+                image_data = tif.asarray()
+                self.view.show_most_recent_image(image_data)
+        except IOError as error:
+            logger.error("Error reading image: %s", error)
+            return
+
+    def display_last_modified_image(self, image_path: str):
+        """Display the last modified image"""
+        if not Path(image_path).exists():
+            return
+        try:
+            with tifffile.TiffFile(image_path) as tif:
+                image_data = tif.asarray()
+                self.view.show_last_modified_image(image_data)
+        except IOError as error:
+            logger.error("Error reading image: %s", error)
+            return
```

Comment:
I don't think this is used. Just a duplicate of `update_image()`

Reply:
Nice catch! this was left over from previous refactoring. have removed.